xml/System.Security.Cryptography/MD5.xml

```diff
@@ -47,7 +47,7 @@
  The <xref:System.Security.Cryptography.HashAlgorithm.ComputeHash%2A> methods of the <xref:System.Security.Cryptography.MD5> class return the hash as an array of 16 bytes.  Note that some MD5 implementations produce a 32-character, hexadecimal-formatted hash.  To interoperate with such implementations, format the return value of the <xref:System.Security.Cryptography.HashAlgorithm.ComputeHash%2A> methods as a hexadecimal value.  
   
 > [!NOTE]
->  Newer hash functions, such as the Secure Hash Algorithms SHA-256 and SHA-512, are available. Consider using the <xref:System.Security.Cryptography.SHA256> class or the <xref:System.Security.Cryptography.SHA512> class instead of the <xref:System.Security.Cryptography.MD5> class. Use <xref:System.Security.Cryptography.MD5> only for compatibility with legacy applications and data.  
+>  Due to collision problems with MD5/SHA1, Microsoft recommends SHA256 or and SHA512. Consider using the <xref:System.Security.Cryptography.SHA256> class or the <xref:System.Security.Cryptography.SHA512> class instead of the <xref:System.Security.Cryptography.MD5> class. Use <xref:System.Security.Cryptography.MD5> only for compatibility with legacy applications and data.  
```

Comment:
```suggestion
>  Due to collision problems with MD5/SHA1, Microsoft recommends SHA256 or SHA512. Consider using the <xref:System.Security.Cryptography.SHA256> class or the <xref:System.Security.Cryptography.SHA512> class instead of the <xref:System.Security.Cryptography.MD5> class. Use <xref:System.Security.Cryptography.MD5> only for compatibility with legacy applications and data.  
```